changelog.md

```diff
@@ -8,6 +8,8 @@
    * Set up for translating the Wings of Victory campaign (PR#4265)
    * Changed the :help command's output to split over multiple lines
    * Added translatable explanations of :droid, :help and :idle's arguments
+ ### WML engine
+   * Ranges can now have upper limit "infinity" instead using a big number like "99" or "99999"
```

Comment:
Instead _of_!

Reply:
Oops - thanks. Will fix on Sunday, unless someone else fixes it first.

Reply:
Fixed in 27209171aff16a8033df4b42ac75c33e902ad425.